ci/workflows/main.yml

```diff
@@ -186,6 +186,22 @@ steps:
     timeout_in_minutes: 5
     retry: *auto-retry
 
+  - label: "unit test"
+    command: "ci/scripts/pr-unit-test.sh"
+    plugins:
+      - ./ci/plugins/swapfile
+      - gencer/cache#v2.4.10: *cargo-cache
+      - seek-oss/aws-sm#v2.3.1:
+          env:
+            CODECOV_TOKEN: my-codecov-token
+      - docker-compose#v3.9.0:
+          run: rw-build-env
+          config: ci/docker-compose.yml
+          environment:
+            - CODECOV_TOKEN
+    timeout_in_minutes: 15
```

Comment:
Have you test that 15mins is enough to finish?

Reply:
yes, usually only seven or eight minutes.